src/main/scala/encry/view/WalletStorageHolder.scala

```diff
@@ -0,0 +1,33 @@
+package encry.view
+
+import akka.actor.Actor
+import scala.concurrent.duration._
+import encry.EncryApp.{settings, system}
+import encry.api.http.routes.GetAllBoxes
+import encry.modifiers.state.box.EncryBaseBox
+import encry.stats.StatsSender.CurrentUtxosQtyInIOdb
+import encry.view.history.EncryHistory
+import encry.view.mempool.EncryMempool
+import encry.view.state.UtxoState
+import encry.view.wallet.EncryWallet
+import scala.concurrent.ExecutionContext.Implicits.global
+import encry.view.EncryNodeViewHolder.ReceivableMessages.GetDataFromCurrentView
+
+class WalletStorageHolder extends Actor {
+
+  var buffer = Seq.empty[EncryBaseBox]
+
+  if (settings.influxDB.isDefined) { system.scheduler.schedule(10 second, 60 second) {
+      system.actorSelection("user/nodeViewHolder") !
+        GetDataFromCurrentView[EncryHistory, UtxoState, EncryWallet, EncryMempool, Seq[EncryBaseBox]] {
+          _.vault.walletStorage.allBoxes }
+    }
+  }
+
+  override def receive: Receive = {
+    case GetAllBoxes() => sender() ! buffer.takeRight(500)
```

Comment:
Are you sure that sender knows how to handle this message that consists of Seq[EncryBaseBox]